spk/inotify-tools/src/service-setup.sh

```diff
@@ -0,0 +1,44 @@
+
+BIN_COMMANDS="inotifywait inotifywatch"
+LIBRARY="libinotifytools.so"
+LIBRARY_MAIN_VERSION="0"
+LIBRARY_SUB_VERSION="4.1"
+LIBRARY_WITH_VERSION="${LIBRARY}.${LIBRARY_MAIN_VERSION}.${LIBRARY_SUB_VERSION}"
+
+service_postinst ()
+{
+    for cmd in $BIN_COMMANDS
+    do
+        if [ -e "${SYNOPKG_PKGDEST}/bin/$cmd" ]; then
+            ln -s "${SYNOPKG_PKGDEST}/bin/$cmd" "/usr/local/bin/$cmd"
+        fi
+    done
+
+    if [ -e "${SYNOPKG_PKGDEST}/lib/$LIBRARY_WITH_VERSION" ]; then
+        ln -s "${SYNOPKG_PKGDEST}/lib/$LIBRARY_WITH_VERSION" "/usr/local/lib/$LIBRARY_WITH_VERSION"
+        ln -s "${SYNOPKG_PKGDEST}/lib/$LIBRARY_WITH_VERSION" "/usr/local/lib/$LIBRARY"
+        ln -s "${SYNOPKG_PKGDEST}/lib/$LIBRARY_WITH_VERSION" "/usr/local/lib/${LIBRARY}.${LIBRARY_MAIN_VERSION}"
+    fi
```

Comment:
When built properly with `prefix` option, executables are expected to search for library in `/var/packages/inotify-tools/target/lib/` thanks to ELF option RPATH.
If it is not correct, it is a best option to fix compilation chain to get it work as expected, instead of linking libraries in `/usr/local/lib` which may conflict with other packages or DSM executables.